components/roadmap-grid/Roadmap.module.css

```diff
@@ -9,7 +9,7 @@
 
 .scrollable {
   extend: grid;
-  height: calc(100vh - 300px);
+  height: calc(100vh - 280px);
```

Comment:
we should probably use react to calculate the height of things above the roadmap so that this is always accurate.

Reply:
do we want to do this as part of this task?

Reply:
we can do it separately. mind creating an issue?

Reply:
created https://github.com/pln-planning-tools/StarMaps/issues/205